back/boxtribute_server/business_logic/box_transfer/agreement/crud.py

```diff
@@ -140,9 +140,9 @@ def create_transfer_agreement(
 ):
     """Insert information for a new TransferAgreement in the database. Update
     TransferAgreementDetail model with given source/target base information. By default,
-    the agreement is established between all bases of both organisations (indicated by
-    NULL for the Detail.source/target_base field). As a result, any base that added to
-    an organisation in the future would be part of such an agreement.
+    the agreement is established with all non-deleted bases of the partner organisation.
+    As a result, any base that added to the partner organisation will NOT be part of the
```

Comment:
We should probably say and will need to establish an additional agreement to transfer goods